### ISSUE TITLE
RPM package 

### ISSUE THREAD

**Issue by mikart143** [vinifmor/bauh#39]
Hello ! 
I am huge fun of your project. It will be very nice to see RPM packages for redhat base system and also opensuse.

**Comment by vinifmor** [comment]
Hello. It is nice to hear that bauh has been useful for you. Do you mean a rpm package to install the application ?

**Comment by mikart143** [comment]
Yes to install bauh on system. For me is must have and for other people I think too.

**Comment by vinifmor** [comment]
I'm currently studying the best approach to distribute bauh to all Linux distros. I will see how difficult is to distribute it in RPM format. A Flatpak version would work for you ?

**Comment by mikart143** [comment]
I think it is good :smile:

**Comment by docop** [comment]
An RPM file can be quite good, as to get support for Centos, with the rpm you can get access to all Centos machine and run much more app vs installer too many 'container'.  Thanks in advance. Is it still in the plan ?

**Comment by vinifmor** [comment]
@docop it is on the backlog. But currently there are other prioritized features. If someone helps on this issue, I can put it on top.

**Comment by AsciiWolf** [comment]
When making a RPM package, please also consider making a DEB package as well. :-)

**Comment by dnkmmr69420** [comment]
Bauh should be able to manage rpms as well
